packages/datadog-plugin-pino/src/index.js

```diff
@@ -2,17 +2,16 @@
 
 const { LOG } = require('../../../ext/formats')
 
-function createWrapPino (tracer, config) {
+function createWrapPino (tracer, config, symbol, wrapper) {
```

Comment:
Not sure this is how you guys usually pass params to `createWrap...()`, I couldn't find an example to replicate. So if there is a "standard" way to do it, let me know.

Reply:
Yes, that is generally how additional params are passed. There isn't a standard per se however, this pattern is used a lot mostly because it's convenient when wrapping but it's not mandatory.

Reply:
Ok, thanks for the precision!